ma/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class MissingUtilityHashError(Exception):
+    def __init__(self, source):
+        message = "No utility hash present, cannot execute. Context: [{}]".format(source)
+        Exception.__init__(self, message)
```

Comment:
Try to use "super": super(Exception, self).__init__(message)

It'll work as long as you use "new-style classes" (read: inherit from `object`).